src/NetscriptFunctions.ts

```diff
@@ -1681,15 +1681,21 @@ export const ns: InternalAPI<NSFull> = {
       }
       return runningScript.onlineExpGained / runningScript.onlineRunningTime;
     },
-  nFormat: (ctx) => (_n, _format) => {
-    const n = helpers.number(ctx, "n", _n);
-    const format = helpers.string(ctx, "format", _format);
-    if (isNaN(n)) {
-      return "";
-    }
-
-    return nFormat(n, format);
-  },
+  nFormat:
+    (ctx) =>
+    (_n, format = {}) => {
+      const n = helpers.number(ctx, "n", _n);
+      // Provide legacy
+      if (typeof format !== "object") {
+        throw helpers.makeRuntimeErrorMsg(
+          ctx,
+          "Incorrect usage of nFormat. The formatOptions for nFormat is no longer a string.\n" +
+            "Please review the latest documentation and adjust usage accordingly.\n" +
```

Comment:
Can we provide a direct link to the docs here?

Reply:
Addressed in d64dfe0